apps/meteor/client/startup/mergeSubscriptionsAndRooms.ts

```diff
@@ -53,6 +53,7 @@ const mergeSubRoom = (subscription: ISubscription): SubscriptionWithRoom => {
 			closedAt: 1,
 			responseBy: 1,
 			priorityId: 1,
```

Comment:
```suggestion
```
Can we remove this param now?

Reply:
Hey hey, remember we'll still asign priorities to rooms 😬 so we won't remove the field.

Reply:
Makes sense :+1: Thanks!